data_safe_haven/infrastructure/project_manager.py

```diff
@@ -286,12 +286,29 @@ def destroy(self) -> None:
             raise DataSafeHavenPulumiError(msg) from exc
 
     def ensure_config(self, name: str, value: str, *, secret: bool) -> None:
```

Comment:
Just wondering whether `ensure` is still the best verb for this. Can you think of a better one @JimMadge ?

Reply:
Ensure is what I would choose, so I might not be the best person to ask :smile: